tests/Issues/NamespacedUse/Fixture/skip_different_letter_case.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Rector\Tests\Issues\NamespacedUseAutoImport\Fixture;
+
+use App\Bar\JsoN;
+
+final class Json{}
```

Comment:
this refer fatal error on the first place, can be removed:

```suggestion
```

Ref https://3v4l.org/SbeLH